mhctools/base_commandline_predictor.py

```diff
@@ -20,9 +20,7 @@
 
 from six import string_types
 from typechecks import require_string, require_integer, require_iterable_of
-from mhcnames import normalize_allele_name
-from mhcnames.parsing_helpers import AlleleParseError
-
+from mhcnames import normalize_allele_name, AlleleParseError
```

Comment:
Sorry, GitHub crapping out on me, same comment as above that you already replied to.